packages/common/src/apolloUtils.ts

```diff
@@ -287,11 +276,34 @@ export const errorLink = onError(({ graphQLErrors, networkError }) => {
         // eslint-disable-next-line no-console
         console.error(`GQL_ERROR: ${JSON.stringify(error, null, 2)}`);
       }
-    }
-
-    if (networkError) {
+    } else if (networkError) {
       // eslint-disable-next-line no-console
       console.error(`NETWORK_ERROR: ${JSON.stringify(networkError, null, 2)}`);
     }
   }
 });
+
+function toastNetworkError(error: Error | ServerParseError | ServerError) {
+  const statusCode = "statusCode" in error ? error.statusCode : null;
+  // ignore 403 errors: these are typically caused by CSRF token
+  // admin ui has no middleware that could redirect to backend when it's missing
+  if (statusCode != null && statusCode === 403) {
+    return;
+  }
```

Comment:
Shouldn't we still show an error for 403 errors, so user gets some feedback from them 🤔

Reply:
Can be simplified to just `statusCode === 403`?